settings.gradle.kts

```diff
@@ -12,21 +12,21 @@ dependencyResolutionManagement {
         create("libs") {
             library("springdoc-openapi-ui", "org.springdoc:springdoc-openapi-ui:1.6.9")
 
-            library("junit-jupiter", "org.junit.jupiter:junit-jupiter:5.8.2")
-            library("junit-jupiter-engine", "org.junit.jupiter:junit-jupiter-engine:5.8.2")
-            library("junit-jupiter-api", "org.junit.jupiter:junit-jupiter-api:5.8.2")
+            library("junit-jupiter", "org.junit.jupiter:junit-jupiter:5.9.0")
+            library("junit-jupiter-engine", "org.junit.jupiter:junit-jupiter-engine:5.9.0")
+            library("junit-jupiter-api", "org.junit.jupiter:junit-jupiter-api:5.9.0")
 
             library("moshi-kotlin", "com.squareup.moshi:moshi-kotlin:1.13.0")
             library("moshi-adapters", "com.squareup.moshi:moshi-adapters:1.13.0")
 
-            library("swagger-jaxrs2-jakarta", "io.swagger.core.v3:swagger-jaxrs2-jakarta:2.2.0")
-            library("swagger-gradle-plugin", "io.swagger.core.v3:swagger-gradle-plugin:2.2.0")
-            library("swagger-annotations", "io.swagger.core.v3:swagger-annotations:2.2.0")
+            library("swagger-jaxrs2-jakarta", "io.swagger.core.v3:swagger-jaxrs2-jakarta:2.2.2")
+            library("swagger-gradle-plugin", "io.swagger.core.v3:swagger-gradle-plugin:2.2.2")
+            library("swagger-annotations", "io.swagger.core.v3:swagger-annotations:2.2.2")
 
-            library("okhttp", "com.squareup.okhttp3:okhttp:4.9.3")
+            library("okhttp", "com.squareup.okhttp3:okhttp:4.10.0")
             library("rs-api", "jakarta.ws.rs:jakarta.ws.rs-api:3.1.0")
 
-            library("skyminder-client", "org.dataland:skyminder-client:0.1.6")
+            library("skyminder-client", "org.dataland:skyminder-client:0.1.7")
             library("dataland-edc-client", "org.dataland:dataland-edc-client:0.2.3")
```

Comment:
0.2.5

Reply:
Ja. Aber 0.2.5 ist dann auch schon mit der neuen EDC-Version. Das ist ja noch on hold und wird dann in DALA-297 gemacht oder nicht?

Reply:
ok